can_tools/scrapers/variables.py

```diff
@@ -2,31 +2,31 @@
 
 from can_tools.scrapers import CMU
 
-INITIATING_VACCINATIONS_ALL = CMU(
+INITIATING_VACCINATIONS_ALL: CMU = CMU(
```

Comment:
Is this necessary when the variable is directly assigned?

Reply:
For some reason my editor wasn't picking up that it was a CMU instance when I was using `variables.INITIATING_VACCINATIONS_ALL`

I agree it shouldn't be necessary, but I guess for my tooling it was?